app/helpers/application_helper.rb

```diff
@@ -139,4 +139,14 @@ def link_to_library_website_search
   def search_type_name
     t("searchworks.search_dropdown.#{controller_name}.label")
   end
+
+  # TODO: remove after migration is complete
+  def folio_disclaimer(document)
+    return false unless document['folio_json_struct'] && document['marc_json_struct']
+
+    marc_source = JSON.parse(document['folio_json_struct']&.first)&.dig('instance', 'source') == 'MARC'
+    leader = JSON.parse(document['marc_json_struct']&.first)&.dig('leader')
+    # currently FOLIO MARC records have a leader that starts with 8 spaces
+    marc_source && leader&.match(/^ {8}/)
```

Comment:
I think we could use the already-parsed MARC record instead of re-parsing it?

```suggestion
    marc_source  && document.to_marc&.leader&.match(/^ {8}/)
```